Utils/Dataflow/019_esFormat/run.sh

```diff
@@ -1,23 +1,32 @@
-#/usr//bin/env bash
+#/usr/bin/env bash
 
 base_dir=$(cd "$(dirname "$(readlink -f "$0")")"; pwd)
 
+ES_CONFIG=$base_dir/../../Elasticsearch/config/es
+CONFIG_DEFAULT=TRUE
+
 usage() {
-  echo "USAGE:
-$(basename "$0") [-c CONFIG] [--] [ARGS]
+  echo "usage: $(basename "$0") [-h] [-c CONFIG] [--] [ARGS]
   
-PARAMETERS:
-  CONFIG -- configuration file
-  ARGS   -- arguments to be passed to the PHP script
-"
-}
+optional arguments:
+  -h, --help   show this help message and exit
 
-ES_CONFIG=$base_dir/../../Elasticsearch/config/es
+  -c CONFIG    configuration file with parameters for
+               accessing data in Elasticsearch
+               Default value: $ES_CONFIG
+
+ARGS, arguments to be passed to the PHP script:"
+# Display part of esFormat.php's help describing its arguments.
+$base_dir/esFormat.php -h 2>&1 | sed 1,5d >&2
```

Comment:
This version relies on current formet of the `esFormat`'s help message, and should it change -- this part must be changed as well. And I bet the one changing help message won't remember to do it ;)

Current version works, so it can be merged as-is, but it's better not to leave such traps behind. If `sed` can't skip plus one line after `/--help/` (I mean, in simple way) -- maybe `awk` will do better.

---

*If it's difficult to fix right now, it's fine to leave as-is (but add ToDo note, Trello card, or something else as a reminder).*

Reply:
Added note, 5cf279d.